src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -104,6 +115,30 @@ void CaptureViewElement::OnDrag(int x, int y) {
   RequestUpdate();
 }
 
+bool CaptureViewElement::IsMouseOver(const Vec2& mouse_pos) {
+  if (parent_ != nullptr && !parent_->IsMouseOver(mouse_pos)) {
```

Comment:
I don't think that this is right. IsMouseOver should return true or false independently of the parent.

Reply:
Not sure if there is a "right" or "wrong" here, it's a design choice, and we can discuss it. My reasoning: Without this check, mouse events and `IsMouseOver` are not consistent. Mouse events bubble down to the children, so if the mouse is outside of the parent, it will not pass the mouse event to any children. I think it's weird to have `IsMouseOver` return `true`, but not have any events (mouse wheel, e.g.) reach the child.

Reply:
I've added a comment to the method. We now also have `ContainsPoint()` which only checks the element itself and not the parent, with this both functionalities are offered - WDYT?

Reply:
ok, you've already added. If everyone else is fine with it, I accept it :)